src/components/calcite-accordion/calcite-accordion.e2e.ts

```diff
@@ -1,19 +1,24 @@
 import { newE2EPage } from "@stencil/core/testing";
-import { renders } from "../../tests/commonTests";
+import { accessible, renders } from "../../tests/commonTests";
 
 describe("calcite-accordion", () => {
-  it("renders", async () => renders("calcite-accordion"));
-
-  it("renders default props when none are provided", async () => {
-    const page = await newE2EPage();
-    await page.setContent(`
-    <calcite-accordion>
+  const accordionContent = `
     <calcite-accordion-item item-title="Accordion Title 1" id="1">Accordion Item Content
     </calcite-accordion-item>
     <calcite-accordion-item item-title="Accordion Title 1" id="2" active>Accordion Item Content
     </calcite-accordion-item>
     <calcite-accordion-item item-title="Accordion Title 3" id="3">Accordion Item Content
     </calcite-accordion-item>
+ `;
+  it("renders", async () => renders("calcite-accordion"));
+
+  it("is accessible", async () => accessible(`<calcite-accordion>${accordionContent}</calcite-accordion>`));
```

Comment:
Being pedantic here, but new tests would warrant its own `test` type commit.

Reply:
I’ll split the rest of these upcoming pr into separate story / test pr 👍